plutus-cbor/test/Plutus/Codec/CBOR/EncodingSpec.hs

```diff
@@ -71,46 +139,53 @@ propCompareWithOracle ::
   x ->
   Property
 propCompareWithOracle encodeOracle encodeOurs x =
-  ( convert (CBOR.toStrictByteString oracle) === ours
+  ( Plutus.toBuiltin (CBOR.toStrictByteString oracle) === ours
   )
     & counterexample ("value: " <> show x)
     & counterexample ("\n─── cborg: \n" <> CBOR.prettyHexEnc oracle)
-    & counterexample ("\n─── ours: \n" <> toString (encodeBase16 $ convert ours))
+    & counterexample ("\n─── ours: \n" <> toString (encodeBase16 $ Plutus.fromBuiltin ours))
  where
   oracle = encodeOracle x
   ours = encodingToBuiltinByteString (encodeOurs x)
 
 -- | Measure that the execution cost of encoding a certain value 'x' is small in
 -- front of some max execution budget.
 propCostIsSmall ::
```

Comment:
Suggestion: `small` seems an absolute statement, and does not appear to really be true? I would rather name this property `propCostIsSmallerThan` and get rid of the comment.